app/javascript/courses/review/components/CoursesReview__ChecklistShow.res

```diff
@@ -179,13 +191,17 @@ let make = (~reviewChecklist, ~feedback, ~updateFeedbackCB, ~showEditorCB, ~canc
     </div>
     <div
       className="flex justify-end bg-white md:bg-gray-100 border-t sticky bottom-0 px-4 md:px-6 py-2 md:py-4 mt-4">
-      <button
-        className="btn btn-primary w-full md:w-auto"
-        disabled={selection->ArrayUtils.isEmpty}
-        onClick={_ =>
-          generateFeedback(checklist, selection, feedback, setSelecton, updateFeedbackCB)}>
-        {t("generate_feedback_button")->str}
-      </button>
+      {switch SubmissionDetails.reviewer(submissionDetails) {
```

Comment:
This probably adds a bug to the existing behaviour. On a reviewed submission, trying adding another feedback using the _Add Another Feedback_ button. You can see that the _Generate Feedback_ button will be missing.

Reply:
Updated the logic @mahesh-krishnakumar . I think this change will do the needful.